internal/security/secretstore/init.go

```diff
@@ -123,45 +138,46 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 				shouldContinue = false
 			case http.StatusNotImplemented:
 				lc.Info(fmt.Sprintf("vault is not initialized (status code: %d). Starting initialization and unseal phases", sCode))
-				_, err := vc.Init(configuration.SecretService.VaultSecretThreshold,
-					configuration.SecretService.VaultSecretShares, &initResponse)
-				if configuration.SecretService.RevokeRootTokens {
+				initResponse, err = client.Init(secretStoreConfig.VaultSecretThreshold, secretStoreConfig.VaultSecretShares)
```

Comment:
Not sure since that is how existing code is. I just adjusted for new func signature. Agree it needs to be handled. Looking closer to determine best approach.

Reply:
ok, and also the err in line 147 `client.Unseal` is potentially overwritten by `vmkEncryption.EncryptInitResponse` too : I suggest using different error variable name and handled it properly.

Reply:
@bnevis-i, please review these changes I have made to here. THX!

Reply:
> also the err in line 147 `client.Unseal` is potentially overwritten

Also addressed in latest changes.